### ISSUE TITLE
MultiComboBox: regularly failing test

### ISSUE THREAD

**Issue by ilhan007** [SAP/ui5-webcomponents#2845]
**Describe the bug**
Hello @SAP/ui5-webcomponents-topic-rl 
There is a test case that fails regularly on the Travis, on the central build,  and slows down the development process.
The test case is "aria-describedby value according to the tokens count". Please investigate what might not be robust enough, what would make it fail. It is ok if you decide to remove some parts and make it simpler. I am setting its priority to High, because it has significant impact on the review/merge process.

When it fails the error is:
**aria-describedby text is correct: expected 'Contains 2 tokens' to equal 'Contains 1 token'**
<img width="868" alt="Screenshot 2021-02-19 at 1 44 01 PM" src="https://user-images.githubusercontent.com/15702139/108507701-5e3a4b80-72c3-11eb-8d57-c68a7e36d992.png">
